### PR TITLE
Improve docker docs

### PR DESCRIPTION
**Pull request by melekes**
Fixes https://github.com/tendermint/tendermint/issues/883

### CODE REVIEW

File: docker/README.rst [hunk 1]
```diff
@@ -1,120 +1,71 @@
...
 
-    docker build -t ethermint ethermint
+::
 
-The application configuration will be stored at ``/ethermint``. The
-files required for initializing ethermint (the files in the source
-``setup`` folder) are under ``/setup``.
+    mkdir /ethermintdata
+    wget -O /ethermintdata/genesis.json https://github.com/tendermint/ethermint/raw/master/setup/genesis.json
+    docker run --rm -v /ethermintdata:/ethermint tendermint/ethermint ethermint --datadir /ethermint init /ethermint/genesis.json
```
Comment (zramsay) [inline docker/README.rst]: i keep getting `Fatal: Failed to read genesis file: open /ethermint/genesis.json: no such file or directory` on running this last line.

Comment (melekes) [reply docker/README.rst]: damn, I thought I've fixed that

Comment (melekes) [reply docker/README.rst]: I've just replayed all the steps. All works for me. Did you download `wget -O /ethermintdata/genesis.json https://github.com/tendermint/ethermint/raw/master/setup/genesis.json` genesis file?

Comment (melekes) [reply docker/README.rst]: whats the output of: 

```
ls -la /ethermintdata
GENESIS FILE SHOULD BE THERE
docker run --rm -v /ethermintdata:/ethermint tendermint/ethermint ls -la /ethermint
GENESIS FILE SHOULD BE THERE
```

Comment (zramsay) [reply docker/README.rst]: works now!

**Pull request merged**